examples/satellite/task-polling/suite.rc

```diff
@@ -1,10 +1,12 @@
 #!Jinja2
 
-title = Demonstrates real time satellite data processing
+title = Real time satellite data processing demo, variant 1 of 3
+
 description = """
-Each successive integer cycle retrieves and processes the next
-arbitrarily timed and arbitrarily labelled dataset, in parallel
-with previous cycles if the data comes in quickly."""
+Successive cycle points retrieve and processe the next arbitrarily timed and
+labelled dataset, in parallel if the data comes in quickly. This variant of the
```

Comment:
No, the meaning is: datasets will be processed in parallel if they come in quickly enough, otherwise not.